### PR TITLE
Add locale for es-CO

### PR DESCRIPTION
**Pull request by jorgeegomez**
I included the tests files, although I don't know how to run the snapshot tests. The output seems correct against past calendars.

### CODE REVIEW

**Comment by lauft** [comment]
Closes #13

File: holidata/holidays/es-CO.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    12-25: [NRF] Navidad
+    3 days before Easter: [NRV] Jueves Santo
+    2 days before Easter: [NRV] Viernes Santo
+    Easter: [NRV] Domingo de Pascua
+    """
+
+    locale = "es-CO"
+    easter_type = EASTER_WESTERN
+
+    def holiday_ascencion(self):
```
[the remark this answers is not in the document]
Comment (jorgeegomez) [reply holidata/holidays/es-CO.py]: You're right, I was focusing on the literal rule: "first monday after...", without realizing that Easter is a Sunday always.

File: tests/snapshots/snap_test_holidata.py [hunk 2]
```diff
@@ -630,3 +631,17 @@
...
+snapshots['test_holidata_produces_holidays_for_locale_and_year[es_CO-2012] 1'] = FileSnapshot('snap_test_holidata/test_holidata_produces_holidays_for_locale_and_year[es_CO-2012] 1.py')
+
+snapshots['test_holidata_produces_holidays_for_locale_and_year[es_CO-2013] 1'] = FileSnapshot('snap_test_holidata/test_holidata_produces_holidays_for_locale_and_year[es_CO-2013] 1.py')
+
+snapshots['test_holidata_produces_holidays_for_locale_and_year[es_CO-2014] 1'] = FileSnapshot('snap_test_holidata/test_holidata_produces_holidays_for_locale_and_year[es_CO-2014] 1.py')
+
+snapshots['test_holidata_produces_holidays_for_locale_and_year[es_CO-2015] 1'] = FileSnapshot('snap_test_holidata/test_holidata_produces_holidays_for_locale_and_year[es_CO-2015] 1.py')
+
+snapshots['test_holidata_produces_holidays_for_locale_and_year[es_CO-2016] 1'] = FileSnapshot('snap_test_holidata/test_holidata_produces_holidays_for_locale_and_year[es_CO-2016] 1.py')
+
```
Comment (lauft) [inline tests/snapshots/snap_test_holidata.py:646]: I am missing the snapshots for 2017 - 2020... 🤔

Comment (lauft) [reply tests/snapshots/snap_test_holidata.py:646]: ... Found them. way up... 😄

**Comment by lauft** [comment]
> `tests/snapshots/snap_test_holidata.py` is missing the snapshots for 2017 - 2020...

Found them. I need to sort the `snap_test_holidata.py` file... 😆

**Pull request merged**